package.json

```diff
@@ -93,7 +93,7 @@
     "react-native": "0.63.4",
     "react-native-background-timer": "2.4.1",
     "react-native-ble-plx": "2.0.2",
-    "react-native-blockies-svg": "Emurgo/react-native-blockies-svg",
+    "react-native-blockies-svg": "git+https://github.com/Emurgo/react-native-blockies-svg",
```

Comment:
what does this do?

Reply:
When adding modules that use git protocol over yarn/npm adding this way makes it compatible with both.

Reply:
why is this necessary?

Reply:
The project must be agnostic about the package tool, before it was tight to yarn.